### PR TITLE
Implemented code for static UI of select facility screen (#1x69ajp)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

**Comment by azkyakhan** [comment]
select-facility
e08ea15be.png)

set-safety-stock-modal
select-facility-csv-upload
66594a.png)

**Review by dt2patel — CHANGES_REQUESTED** [review]
Some changes

File: src/views/SelectFacility.vue [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+  },
+});
+</script>
+
+<style scoped>
+.filters {
+  border-right: 1px solid var(--ion-color-medium);
+}
+
+ion-item-divider > ion-checkbox[slot="end"] {
```
[the remark this answers is not in the document]
Comment (azkyakhan) [reply src/views/SelectFacility.vue]: Removed it sir

File: src/components/SafetyStockModal.vue [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content>
+    <ion-item>
+      <ion-label>{{ $t("Safety stock") }}</ion-label>
+      <ion-input placeholder="new safety stock" />
+    </ion-item>
+
+    <ion-item lines="none">
```
[the remark this answers is not in the document]
Comment (azkyakhan) [reply src/components/SafetyStockModal.vue]: Done

File: src/views/SelectFacility.vue [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+              </ion-item>
+              <ion-item>
+                <ion-label>Store name</ion-label>
+                <ion-checkbox slot="end" />
+              </ion-item>
+            </ion-card>
+            <ion-card>
+              <ion-item-divider>
+                Store Group
+                <ion-checkbox slot="end"/>
```
Comment (dt2patel) [inline src/views/SelectFacility.vue:94]: @azkyakhan When improving code it's important to have a wholistic approach. You shouldn't just edit the exact comment I made. Instead you have to be proactive about the improvements. 

If you remove slot end selector from css you should naturally remove it from markup too.

**Review by dt2patel — CHANGES_REQUESTED** [review]
Previous fix did not cover all instances of issue.

File: src/views/SelectFacility.vue (hunk 3 above)
Comment (azkyakhan) [reply src/views/SelectFacility.vue:94]: If I am removing `end` slot from markup, Its looking like this - 
checkbox-issue

that is why I havenot removed it from markup after removing from selector.

Comment (dt2patel) [reply src/views/SelectFacility.vue:94]: Thanks for the explanation. Next time you can instead reference documentation and mention support for slots in ion-item dividers. That is more helpful: https://ionicframework.com/docs/api/item-divider#slots

**Review by dt2patel — APPROVED** [review]

**Pull request merged**